core/src/main/java/com/datastax/oss/driver/internal/core/metrics/DropwizardMetricsFactory.java

```diff
@@ -73,6 +99,23 @@ public DropwizardMetricsFactory(InternalDriverContext context) {
     }
   }
 
+  @VisibleForTesting
+  static Duration getAndValidateEvictionTime(DriverExecutionProfile config, String logPrefix) {
+    Duration evictionTime = config.getDuration(DefaultDriverOption.METRICS_NODE_EXPIRE_AFTER);
+
+    if (evictionTime.compareTo(LOWEST_ACCEPTABLE_EXPIRE_AFTER) < 0) {
+      LOG.warn(
+          "[{}] Value too low for {}: {} (It should be higher than {}). Forcing to {} instead.",
+          logPrefix,
+          DefaultDriverOption.METRICS_NODE_EXPIRE_AFTER.getPath(),
+          evictionTime,
+          LOWEST_ACCEPTABLE_EXPIRE_AFTER,
+          DEFAULT_EXPIRE_AFTER);
```

Comment:
Yes, I was thinking about it but on the other hand, if someone provides the wrong value maybe it is better to just set it to default?

Reply:
If someone overrides the default value to, say, 1 minute, it's probably because they really want an aggressive eviction policy. I think it would make sense to provide them with the most aggressive interval that we support, so 5 minutes, not 1 hour.

Reply:
It does make sense. Done